core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-editor-modal.controller.ts

```diff
@@ -51,13 +51,16 @@ angular.module('oppia').controller('QuestionSuggestionEditorModalController', [
     $scope.questionId = questionId;
     $scope.skill = skill;
     $scope.skillDifficulty = skillDifficulty;
-    $scope.skillDifficultyString = Object.entries(
-      SKILL_DIFFICULTY_LABEL_TO_FLOAT).find(
-      entry => entry[1] === skillDifficulty)[0];
     $scope.misconceptionsBySkill = {};
     $scope.misconceptionsBySkill[$scope.skill.getId()] = (
       $scope.skill.getMisconceptions());
     ContextService.setImageSaveDestinationToLocalStorage();
+    $scope.getDifficultyString = function(skillDifficulty) {
```

Comment:
I think this would make more sense as `setDifficultyString`, as we're setting the `skillDifficultyString` variable in the scope.